src/Contracts/Services/MediaServiceContract.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace RonasIT\Media\Contracts\Services;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Pagination\LengthAwarePaginator;
+
+interface MediaServiceContract
+{
+    public function search(array $filters): LengthAwarePaginator;
+
+    public function create(string $content, string $fileName, array $data): Model;
+
+    public function bulkCreate(array $data): array;
+
+    /**
+     * @param $where array|integer|string
+     * @return int
+     */
+    public function delete($where): int;
```

Comment:
please add `exists` method as it using in `DeleteRequest`

Reply:
It seems that this is no longer necessary, since in DeleteMediaRequest.php `get` is now used instead of `exists`.

Reply:
Done.